### DISCUSSION TITLE
Prevent Item Drop/Spawn

### DISCUSSION THREAD

**Discussion by WinstonHenry** [Pryaxis/TShock#2505] [General]
Hi everyone,

is it possible to implement prevention of an item drop, instead of the current item ban logic, where you get "stuck" when trying to use the item, please?

In other words, according to the item ban list, prevent items from dropping/spawning completely.
For ex, if I banned "gel", if I kill a slime, gel shouldn't drop.
Also, if I banned "wooden armor", prevent crafting of the item.

Thanks

**Comment by hdseventh** [comment]
I think you can modify DisableDrops plugin to do this.

**Reply by hdseventh** [reply]
Ah wait nvm, i thought it was when player trying to drop something.
